sys/arm64/cheri/cheri_coaccept.S

```diff
@@ -216,10 +216,18 @@ SIGCODE(switcher_coaccept)
 	 */
 	sub	x11, x4, x13
 	cmp	x11, #0
+	mov	x14, x13
 	bge	2f
 	mov	x13, x4
+	mov	x14, x13
 
 2:
+	//If we use this mov instruction instead of the two "mov x14, x13" above
+	//then this function returns '8' even if x13 and x4 contain values
+	//larger than eight. This is an odd bug that cannot be explained by a
+	//flaw in the assembly code. This issue occurs on qemu. I haven't
+	//tested if it can be reproduced on an actual Morello board.
+	//mov	x14, x13
```

Comment:
Could you achieve the same effect with another label after this `mov` for the backwards jump 4 instructions down?